app/src/main/java/com/codesoom/assignment/CrudHttpHandler.java

```diff
@@ -0,0 +1,149 @@
+package com.codesoom.assignment;
+
+import com.codesoom.HttpEnum.HttpMethodCode;
+import com.codesoom.HttpEnum.HttpStatusCode;
+import com.codesoom.models.Task;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.net.HttpHeaders;
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.io.InputStreamReader;
+import java.io.ByteArrayOutputStream;
+
+import java.net.URI;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+public class CrudHttpHandler implements HttpHandler {
+
+    private Map<Long, Task> tasks = new HashMap<>();
+    private ObjectMapper mapper = new ObjectMapper();
+    private Long autoId = 1L; // 아이디를 자동으로 부여하기 위한 변수
+
+
+    @Override
+    public void handle(HttpExchange exchange) throws IOException {
+
+        String method = exchange.getRequestMethod();
+        URI uri = exchange.getRequestURI();
+
+        String path = uri.getPath();
```

Comment:
이렇게 값을 재할당 할 일이 없는 값은 상수로 선언해 봅시다. 이렇게 하면 어떤 장점이 있을까요?
```suggestion
        final String method = exchange.getRequestMethod();
        final URI uri = exchange.getRequestURI();

        final String path = uri.getPath();
```

Reply:
상수로 선언되기 때문에 임의로 값을 변경하지 못하기 때문에 처리에 중요한 Http Method, uri, path 에 값을 보존하는 효과를 얻을 수 있을 것 같습니다.